### DISCUSSION TITLE
[ODIN][ERROR] - Error: unable to resolve user name to user id

### DISCUSSION THREAD

**Discussion by kaxonomy** [mbround18/valheim-docker#540] [Help!] [answered]
I followed the basic install instructions on Debian 11 but the only thing that appears in the valheim_server.log is '[ODIN][ERROR] - Error: unable to resolve user name to user id' and the server doesn't come online.

**Comment by kaxonomy** [comment, marked as answer]
I'm on Odin 1.5.2 and odin status reports this:
[ODIN][ERROR] - Failed to request server information!
[ODIN][INFO]  - Name: Unknown
[ODIN][INFO]  - Players: 0/0
[ODIN][INFO]  - Map: Unknown
[ODIN][INFO]  - BepInEx Enabled: false

**Reply by mbround18** [reply]
@kaxon-dev try `odin status --local`

**Reply by kaxonomy** [reply]
`odin status --local` provided the same result:
[ODIN][ERROR] - Failed to request server information!
[ODIN][INFO]  - Name: Unknown
[ODIN][INFO]  - Players: 0/0
[ODIN][INFO]  - Map: Unknown
[ODIN][INFO]  - BepInEx Enabled: false

**Reply by mbround18** [reply]
@kaxon-dev are you running this in the same container?

**Reply by kaxonomy** [reply]
Unsure what you mean by that. Odin and Huginn are located in the user's /home/ directory. The installation folder is /home/user/Valheim. I'm running `odin install`, `odin configuration`, `odin start`, and `odin status --local` all inside that Valheim folder.

**Reply by kaxonomy** [reply]
I'm also using
`alias odin="$HOME/odin"`
`alias huginn="$HOME/huginn"`
to identify the command paths. Not sure if putting them in a bin folder and add them to path normally makes a difference.

**Reply by mbround18** [reply]
Either should work, which release are you running from?

**Reply by kaxonomy** [reply]
1.5.2

**Reply by mbround18** [reply]
Just to double check, the server is running when you run something like top or htop?

**Reply by kaxonomy** [reply]
Unsure, but the lack of logs led me to believe it wasn't starting up. I got frustrated and decided to switch to Ubuntu 20 which fixed my problems. I appreciate the support though.
